lib/Mojo/WebSocketProxy/Backend/ConsumerGroups.pm

```diff
@@ -112,6 +118,14 @@ sub timeout {
     return shift->{timeout} //= RESPONSE_TIMEOUT;
 }
 
+=head2 category_timeout_config
+
```

Comment:
please add a line about the subroutine and its default value and what it returns

Reply:
Why there are 2 same pod section ?  https://github.com/binary-com/perl-Mojo-WebSocketProxy/pull/74/files#diff-9e7d49105b98b317451e638ee705edadfafb05e3a9925b625ec533a3f2fd7bafR57